great_ape_safe/ape_api/convex.py

```diff
@@ -32,6 +35,7 @@ def __init__(self, safe):
         # frax contract section
         self.frax_booster = safe.contract(registry.eth.convex.frax.booster)
         self.frax_pool_registry = safe.contract(registry.eth.convex.frax.pool_registry)
+        self.VAULT_TYPES = VAULT_TYPES
```

Comment:
```suggestion

        # dictionary holding the different frax vault pids
        self.VAULT_TYPES = {
            "vafrax": 0,
            "univ2_temple": 1,
            "afrax": 2,
            "curve_lp": 3,
            "badger_fraxbp": 35,
}
```

Reply:
pushed on 8e6b26ccd4cc7b8009980f1e3f269cfd4165facb for formatting